src/ORM/Rule/IsUnique.php

```diff
@@ -77,7 +77,7 @@ protected function _alias($alias, $conditions)
     {
         $aliased = [];
         foreach ($conditions as $key => $value) {
-            $aliased["$alias.$key"] = $value;
+            $aliased["$alias.$key IS"] = $value;
```

Comment:
The query builder seems to avoid creating `title IS 'First Article' ` conditions - is that a feature one can rely on?